src/handler/http/request/organization/settings.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Zinc Labs Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::common::meta::organization::OrganizationSettingResponse;
+use crate::common::utils::json;
+use crate::service::db::organization::get_org_setting;
+use crate::{
+    common::meta::{self, organization::OrganizationSetting},
+    service::db::organization::set_org_setting,
+};
+use actix_web::{get, http, post, web, HttpResponse};
+use std::io::Error;
+
+/** Organization specific settings */
+#[utoipa::path(
+    context_path = "/api",
+    tag = "Organizations",
+    operation_id = "OrganizationSetting",
+    security(
+        ("Authorization"= [])
+    ),
+    params(
+        ("org_id" = String, Path, description = "Organization name"),
+    ),
+    request_body(content = OrganizationSetting, description = "Organization settings", content_type = "application/json"),
+    responses(
+        (status = 200, description="Success", content_type = "application/json", body = HttpResponse),
+        (status = 400, description="Failure", content_type = "application/json", body = HttpResponse),
+    )
+)]
+#[post("/{org_id}/settings")]
+async fn create(
```

Comment:
The function name maybe rename to `settings`?

Reply:
I did it because the full path looked like this in the rest of the code, else it would look like `settings::settings`
```rust
.service(organization::settings::get)
.service(organization::settings::create)
```